docs/astro/src/content/docs/guide/platforms/mobile/android.mdx

```diff
@@ -0,0 +1,198 @@
+---
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+title: Android
+description: Android Platform Guide
+---
+
+// cSpell: ignore xbuild textfields
+
+import LangRefLink from '@slint/common-files/src/components/LangRefLink.astro';
+import { Image } from 'astro:assets';
+
+:::note[Note]
+When developing Slint applications for Android, you can only use Rust as the programming language.
+:::
+
+Also see the documentation of the <LangRefLink lang="rust-slint" relpath="android/">android module in our Rust API documentation</LangRefLink>.
+
+## Project Setup
+
+Slint uses the [android-activity crate](https://github.com/rust-mobile/android-activity) as the interface to
+the operating system, which is re-exported as `slint::android::android_activity`. To get started, follow these steps:
+
+First, your project needs to be a library crate. Add the following to your `Cargo.toml`:
+
+```toml
+[lib]
+crate_type = ["cdylib"]
+```
+
+You also need to select the version of android-activity you want to use:
+
+```toml
+[dependencies]
+slint = { version = "*", features = ["backend-android-activity-06"] }
```

Comment:
When folks copy and paste from here, we **do** want them to at least include the major version - otherwise there will be uncomfortable awakenings when we release a new major version :-). We could just code "1" here, but in other parts of the Rust docs I think we use major.minor. @ogoffart what do you think?

```suggestion
slint = { version = "1.15", features = ["backend-android-activity-06"] }
```